code/modules/surgery/organs/tails.dm

```diff
@@ -105,3 +105,24 @@
 		H.dna.features["tail_monkey"] = "None"
 		H.dna.species.mutant_bodyparts -= "tail_monkey"
 		H.update_body()
+
+/obj/item/organ/tail/lizard/kobold
```

Comment:
don't forget to add a limbgrower design for a /obj/item/organ/tail/lizard/kobold/fake subtype of these

Reply:
please stop suggesting extra features to prs